service/src/main/resources/application.yaml

```diff
@@ -221,6 +221,8 @@ fees-register:
   api:
     url: http://fake-fees-register-api
 
+retry:
+  delay: 50
```

Comment:
Maybe bigger? is there exponential backoff?

Reply:
I think it could be exponential if the backoff is changed to
```java
  backoff = @Backoff(
    delayExpression = "#{${retry.delay:1000}}",
    multiplier = 2,
    maxDelay = 900000
  )
```

Reply:
it is for integration test only